packages/lesswrong/server/callbacks/userCallbacks.ts

```diff
@@ -195,3 +198,65 @@ getCollectionHooks("Users").editSync.add(async function usersEditCheckEmail (mod
   }
   return modifier;
 });
+
+getCollectionHooks("Users").editAsync.add(async function subscribeToForumDigest (newUser: DbUser, oldUser: DbUser) {
+  if (
+    isAnyTest ||
+    forumTypeSetting.get() !== 'EAForum' ||
+    newUser.subscribedToDigest === oldUser.subscribedToDigest
+  ) {
+    return;
+  }
+
+  const mailchimpAPIKey = mailchimpAPIKeySetting.get();
+  const mailchimpForumDigestListId = mailchimpForumDigestListIdSetting.get();
+  const { lat: latitude, lng: longitude } = userGetLocation(newUser);
+  const status = newUser.subscribedToDigest ? 'subscribed' : 'unsubscribed';   
+  await fetch(`https://us8.api.mailchimp.com/3.0/lists/${mailchimpForumDigestListId}/members`, {
+    method: 'POST',
+    body: JSON.stringify({
+      email_address: newUser.email,
+      email_type: 'html', 
+      location: {
+        latitude,
+        longitude,
+      },
+      merge_fields: {
+        FNAME: newUser.fullName
+      },
+      status,
+    }),
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: `API_KEY ${mailchimpAPIKey}`,
+    },
+  });
+});
+
+getCollectionHooks("Users").newAsync.add(async function subscribeToDripCampaign(user: DbUser) {
+  if (isAnyTest || forumTypeSetting.get() !== 'EAForum') {
+    return;
+  }
+  const mailchimpAPIKey = mailchimpAPIKeySetting.get();
+  const mailchimpEAForumListId = mailchimpEAForumListIdSetting.get();
+  const { lat: latitude, lng: longitude } = userGetLocation(user);
```

Comment:
This is basically guaranteed to return the placeholders for users who have just been created. We probably shouldn't be telling mailchimp that all our users are at the placeholder. You can check for `known`.

Reply:
Ah, thanks for catching that! Fixed in https://github.com/centre-for-effective-altruism/EAForum/pull/176/commits/8b5f0cfc96998da0ce9d189fd10aa614c646571a